plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/utils.ts

```diff
@@ -8,11 +8,26 @@ export function compressToString(input: string): string {
     return compressed_data.toString('base64')
 }
 
-export function decompressFromString(input: string): string {
+export function decompressFromString(input: string, doubleDecode = false): string {
```

Comment:
The code makes sense but I'm still a little confused why we see this 😅 How does it get double base64 encoded??

Reply:
Yeah, I don't know either... 🤔 so we merge this, the problem goes away, and we never think about it again 🙈